include/radio.hpp

```diff
@@ -42,6 +42,11 @@ namespace bzzz
             
             Radio();
 
+            /**
+             * @brief Begin Serial com
+            */
+           void beginSerial(int baud);
```

Comment:
Minor comment: just add `@param baud baud rate`

Reply:
On it.

Reply:
Added `@param baud baud rate` as suggested.